php/EE/Migration/Executor.php

```diff
@@ -103,13 +101,18 @@ private static function get_migrations_to_execute() {
     }
 
     private static function get_migrations_from_db() {
-        return Migration::all();
+        return array_column( Migration::all(), 'migration' );
     }
 
     private static function get_migrations_from_fs() {
         // array_slice is used to remove . and .. returned by scandir()
-        $migrations = array_slice( scandir( self::MIGRATION_PATH ), 2 );
-        array_walk( $migrations, function( &$migration, $index ) {
+		$migrations = scandir( self::MIGRATION_PATH );
```

Comment:
Remove first two elements using `array_slice`.

Reply:
That is only needed if migrations are executed from outside phar. I've added the condition for that below.